facebook.js

```diff
@@ -1,78 +1,148 @@
+var stad = chrome.i18n.getMessage("stand");
+var per = chrome.i18n.getMessage("per");
```

Comment:
IMHO the easiest would be

```javascript
let emoji_map = {}
if (locale == 'Locale_en_GB') {
    emoji_map = /* emoji_map_en_GB */;
} else if (locale === 'de_DE') {
    emoji_map = /* emoji_map_de_DE */;
}
```
The actual maps can either be defined verbatim or loaded from file.